docs/learn/learn-async-backing.md

```diff
@@ -200,38 +196,127 @@ blocks in advance to fit more data, Polkadot with asynchronous backing can deliv
+does not access the entire state of a parachain but only the values that changed during that block
+and the merkelized hashes of the unchanged values.
+
+### Pipelining
+
+Asynchronous backing is a feature that introduces
+[pipelining](https://www.techtarget.com/whatis/definition/pipelining) to the parachain block
+[generation, backing and inclusion](./learn-parachains-protocol.md). It is analogous to the logical
+pipelining of processor instruction in "traditional" architectures, where some instructions may be
+executed before others are complete. Instructions may also be executed in parallel, enabling
+multiple processor parts to work on potentially different instructions simultaneously.
+
+Bundles of state transitions represented as blocks may be processed similarly. In the context of
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, pipelining aims to increase the
+throughput of the entire network by completing the backing and inclusion steps for different blocks
+at the same time. Asynchronous backing does not just allow for pipelining within a single pipe (or
+core). It lays the foundation for a large number of pipes (~10 cores) to run for the same parachain
+at the same time. In that way we have two distinct new forms of parallel computation: having one
+block backed and one included simultaneously using one core, and having having `n` blocks backed and
+`n` blocks included simultaneously using `n` cores.
+
+### Contextual Execution
+
+Contextual execution refers to the context built by parablock ancestors used by collators to
+generate new parablocks. The difference in execution context between synchronous and asynchronous
+backing is as follows:
+
+In synchronous backing, the entire execution context comes from the latest relay block and most
+recently included parablock from the same parachain.
+
+In asynchronous backing, a baseline execution context is first set up using a parablock's relay
+parent and the most recently included parablock as of that relay parent. The relay parent need not
+be the most recent relay block. Predecessors of the block under construction live in the
+[unincluded segment](#unincluded-segments) placed directly in the runtime of parachain collators
+(i.e. off-chain).
+
+Synchronous backing uses execution context entirely pulled from the relay chain. While asynchronous
+backing augments this with off-chain context from the unincluded segment.
+
+### Unincluded Segments
+
+The purpose of each unincluded segment is two fold:
+
+- Make each parachain aware of when and at what depth it can build blocks that won't be rejected by
+  the relay chain
+- Provide critical context necessary to build parablocks with parent blocks that have yet to be
+  included. The unincluded segment is all about building parablocks.
+
+Unincluded segments are chains of candidate parablocks that have yet to be included in the relay
```

Comment:
I'd recommend the following:
1. Switch the order of paragraphs one and two so that two comes first
2. Get rid of paragraph three here. If not too repetitive, you could move it to the contextual execution section.